File: BEE 1006.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{

        Scanner input = new Scanner(System.in);
        double a, b, c, avg;
        a = input.nextDouble();
        b = input.nextDouble();
        c = input.nextDouble();
        avg = (a*2 + b*3 + c*5)/(2+3+5);
        System.out.printf("MEDIA = %.1f\n", avg);

    }
}

File: BEE 1006.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = parseFloat(lines.shift());

var avg = (a*2 + b*3 + c*5)/(2+3+5);

console.log("MEDIA =", avg.toFixed(1));